Handler/Package.hs

```diff
@@ -16,7 +16,7 @@ import qualified Data.Text.Lazy as LT
 import           Distribution.Package.ModuleForest
 import           Graphics.Badge.Barrier
 import           Control.Lens
-import           Import
+import           Import hiding (empty)
```

Comment:
I couldn't see the reason for that change, so I investigated and found out that it fixes the shadowed name warning triggered by [`parseIdentitiesLiberally`](https://github.com/juhp/stackage-server/blob/master/Handler/Package.hs#L169):
```haskell
Handler/Package.hs:169:9: Warning:
    This binding for 'empty' shadows the existing binding
      imported from 'Import' at Handler/Package.hs:19:1-23
```

It's good to fix the warning, but I don't like hiding `empty` for that. I think it would be right to rename the shadowing `empty` to e.g. `emptyPlainText`. That would describe it's intention better and won't shadow the standard imports.